parsec/core/gui/greet_device_widget.py

```diff
@@ -174,9 +177,34 @@ def _on_button_send_email_clicked(self):
 
     def _on_send_email_success(self, job):
         # In theory the invitation address shouldn't have changed, but better safe than sorry
-        self.invite_addr = job.ret
-        self.button_send_email.setText(_("TEXT_GREET_DEVICE_EMAIL_SENT"))
-        self.button_send_email.setDisabled(True)
+
+        self.invite_addr, email_sent_status = job.ret
+
+        if email_sent_status == InvitationEmailSentStatus.SUCCESS:
+            self.button_send_email.setText(_("TEXT_GREET_DEVICE_EMAIL_SENT"))
+            self.button_send_email.setDisabled(True)
+        else:
+            if email_sent_status == InvitationEmailSentStatus.BAD_RECIPIENT:
+                show_info_copy_link(
+                    self,
+                    "Email bad recipient",
```

Comment:
No hard-coded strings, it should be translated (same thing for all the other ones).

In case you didn't know, you can use
>python setup.py extract_translations

to add the all the translations to the .po file (instead of doing it manually), makes it easier to find the missing ones.